src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -118,14 +119,17 @@ public static void Clear()
     [Fact]
     public static void SetCursorPosition()
     {
-        // Nothing to verify; just run the code.
-        Console.SetCursorPosition(0, 0);
-        Console.SetCursorPosition(1, 2);
-        Assert.Throws<ArgumentOutOfRangeException>("left", () => Console.SetCursorPosition(-1, 100));
-        Assert.Throws<ArgumentOutOfRangeException>("top", () => Console.SetCursorPosition(100, -1));
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || (!Console.IsInputRedirected && !Console.IsOutputRedirected))
+        {
+            // Nothing to verify; just run the code.
+            // On windows, we might end of throwing IOException, since the handles are redirected.
```

Comment:
What did you decide around keeping the behaviors the same between the implementations?